modules/FormModel.js

```diff
@@ -255,38 +411,55 @@
     },
 
     /**
-     * Pushes values from this form model back to the object models it is tracking. This includes invoking the push callbacks from
-     * computed values
-     * @method push
+     * @method isTrackingObjectModel
+     * @return true if this form model is backed by an Object model. That means that at least one object model was bound to an mapping alias.
      */
-    push: function() {
-      _.each(this.__modelConfigs, function(modelConfig) {
-        this.__copyFields(modelConfig.fields, modelConfig.model, this);
-      }, this);
-      _.each(this.__computed, function(computedConfig) {
-        // If a push callback is defined, fire it.
-        if (computedConfig.push) {
-          computedConfig.push.apply(this, [_.pluck(computedConfig.models, 'model')]);
+    isTrackingObjectModel: function() {
+      return _.size(this.__currentObjectModels) > 0;
+    },
+
+    /**
+     * @method isUpdating
+     * @return true if any updates to an object model will immediately copy new values into this form model.
+     */
+    isUpdating: function() {
+      return this.__currentUpdateEvents.length > 0;
+    },
+
+    /**
+     * Will add listeners that will automatically pull new updates from this form's object models.
+     * @param [pullFirst=false] {Boolean} if true, the form model will pull most recent values then start listening
+     * @method startUpdating
+     */
+    startUpdating: function(pullFirst) {
+      if (this.isTrackingObjectModel() && !this.isUpdating()) {
```

Comment:
ahh, isUpdating() is meant to go with startUpdating().  Less sure we need the name change for isUpdating()